tike/tomo.py

```diff
@@ -158,10 +153,12 @@ def _tomo_interface(obj, obj_corner,
     return (obj, obj_corner, probe, th1, v1, h1)
 
 
-def reconstruct(obj=None,
-                probe=None, theta=None, v=None, h=None,
-                line_integrals=None,
-                algorithm=None, niter=0, **kwargs):
+def reconstruct(
+    obj=None,
+    probe=None, theta=None, v=None, h=None,
+    line_integrals=None,
+    algorithm=None, niter=0, **kwargs
```

Comment:
```niter``` is also a algorithm parameter. Should it be passed through algorithm kwargs (except that you only constraint this to iterative methods and omit FBP for example)?

Reply:
This will be addressed with either #15 or #18.